gojira.sh

```diff
@@ -626,11 +635,16 @@ function build {
     "--label KONG_BUILD_TOOLS=$KONG_BUILD_TOOLS"
   )
 
+  ssl_provider=" * OpenSSL:     $OPENSSL  "
+  if [[ ! -z $BORINGSSL ]]; then
```

Comment:
same as suggested previously

```suggestion
  if [[ -n $BORINGSSL ]]; then
```